packages/xo-web/src/xo-app/vm/tab-general.js

```diff
@@ -59,160 +59,178 @@ export default connectStore(() => {
+            </BlockLink>
+          </Col>
+          <Col mediumSize={3}>
+            <h2 className='form-inline'>
+              <Size
+                value={defined(memory.dynamic[1], null)}
+                onChange={memory => editVm(vm, { memory })}
+              />
+              &nbsp;
+              <span>
+                <Icon icon='memory' size='lg' />
+              </span>
             </h2>
-          </BlockLink>
-          <BlockLink to={`/vms/${vm.id}/stats`}>
-            {statsOverview && <XvdSparkLines data={statsOverview} />}
-          </BlockLink>
-        </Col>
-      </Row>
-      {/* TODO: use CSS style */}
-      <br />
-      <Row className='text-xs-center'>
-        <Col mediumSize={3}>
-          {vm.installTime !== null && (
-            <div className='text-xs-center'>
-              {_('created', {
-                date: (
-                  <FormattedDate
-                    day='2-digit'
-                    month='long'
-                    value={vm.installTime * 1000}
-                    year='numeric'
-                  />
-                ),
-              })}
-            </div>
-          )}
-          {vm.power_state === 'Running' ? (
-            <div>
-              <p className='text-xs-center'>
-                {_('started', {
-                  ago: <FormattedRelative value={vm.startTime * 1000} />,
+            <BlockLink to={`/vms/${id}/stats`}>
+              {statsOverview && <MemorySparkLines data={statsOverview} />}
+            </BlockLink>
+          </Col>
+          <Col mediumSize={3}>
+            <BlockLink to={`/vms/${id}/network`}>
+              <h2>
+                {VIFs.length}x <Icon icon='network' size='lg' />
+              </h2>
+            </BlockLink>
+            <BlockLink to={`/vms/${id}/stats`}>
+              {statsOverview && <NetworkSparkLines data={statsOverview} />}
+            </BlockLink>
+          </Col>
+          <Col mediumSize={3}>
+            <BlockLink to={`/vms/${id}/disks`}>
+              <h2>
+                {formatSize(vmTotalDiskSpace)} <Icon icon='disk' size='lg' />
+              </h2>
+            </BlockLink>
+            <BlockLink to={`/vms/${id}/stats`}>
+              {statsOverview && <XvdSparkLines data={statsOverview} />}
+            </BlockLink>
+          </Col>
+        </Row>
+        {/* TODO: use CSS style */}
+        <br />
+        <Row className='text-xs-center'>
+          <Col mediumSize={3}>
+            {installTime !== null && (
+              <div className='text-xs-center'>
+                {_('created', {
+                  date: (
+                    <FormattedDate
+                      day='2-digit'
+                      month='long'
+                      value={installTime * 1000}
+                      year='numeric'
+                    />
+                  ),
                 })}
+              </div>
+            )}
+            {powerState === 'Halted' || powerState === 'Suspended' ? (
```

Comment:
Why don't you add `|| powerState === 'Paused'` instead?

Reply:
The `start_time` is defined for 'Paused' state.

Reply:
Yes, that's why I suggest to simply add `|| powerState === 'Paused'` to the original code:
```js
powerState === 'Running' || powerState === 'Paused'
  ? // startTime
  : // shutdownTime
```